apps/codeforafrica/src/lib/data/rest/index.js

```diff
@@ -37,6 +37,6 @@ export const api = {
 export async function getPageStaticProps(context) {
   const props = await getPageProps(api, context);
   return {
-    props,
+    props: props || {},
```

Comment:
Looks like we need to sort this one out first. Can we create a quick PR to address this?